config/init_db.sql

```diff
@@ -77,3 +83,8 @@ FLUSH PRIVILEGES;
 
 RESET SLAVE ALL;
 RESET MASTER;
+
+# add custom sql here
```

Comment:
instead of adding this, can't we just look for the line in the file that sets the readonly and add our custom commands above that?

Reply:
I thought that would be more risky ... we don't know how would it react if client provide their own int_db.sql and it has some statements which does turns out true for our condition of readonly... Hence I thought this will be much cleaner approach...